client/signup/jetpack-connect/site-url-input.jsx

```diff
@@ -44,41 +49,40 @@ class JetpackConnectSiteURLInput extends Component {
 		this.refs.siteUrl.refs.textField.focus();
```

Comment:
Note: we should use ref callbacks for the refs here and in JetpackConnectMain (see more here: https://facebook.github.io/react/docs/refs-and-the-dom.html)

Otherwise I'm seeing some warnings because refs are not being attached properly: 

![](https://cldup.com/vj-TmD_Zge.png)